about.html

```diff
@@ -0,0 +1,199 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Sprintplex Summer Coding Camp | Giving the youth a chance</title>
+    <link rel="shortcut icon" type="image/jpg" href="./build/images/favicon.ico" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css"
+    />
+    <link rel="stylesheet" href="./build/css/style.css" />
+    <link rel="stylesheet" href="./build/css/main.css" />
+  </head>
+  <body>
+    <header class="header">
+      <div class="header__user-bar">
+        <div class="header__user row">
+          <i class="fa-brands fa-facebook-f"></i>
+          <i class="fa-brands fa-twitter"></i>
+          <p class="header__language">English</p>
+          <p class="header__user-page">My page</p>
+          <p class="header__logout">Logout</p>
+        </div>
+      </div>
+      <nav class="header__navbar row">
+        <a href="index.html" class="header__home-link">
+          <img
+            src="./build/images/sprintplex-c-c.png"
+            alt="Sprintplex coding camp logo"
+            class="header__brand-img"
+          />
+        </a>
+        <ul class="header__nav-list">
+          <li class="header__nav-item">
+            <a href="about.html" class="header__nav-link">About</a>
+          </li>
+          <li class="header__nav-item">
+            <a href="" class="header__nav-link">Program</a>
+          </li>
+          <li class="header__nav-item">
+            <a href="" class="header__nav-link">Join</a>
+          </li>
+          <li class="header__nav-item">
+            <a href="" class="header__nav-link">Sponsor</a>
+          </li>
+          <li class="header__nav-item">
+            <a href="" class="header__nav-link">News</a>
+          </li>
+          <li class="header__nav-item">
+            <a href="" class="header__cta"
+              ><img
+                src="./build/images/sprintplex.jpeg"
+                alt="Sprintplex logo"
+                class="header__cta-logo"
+            /></a>
+          </li>
+        </ul>
+        <div class="header__hamburger">
```

Comment:
- [ ] The hamburger menu is on the left side, please kindly fix that 👍

Your design
<img width="469" alt="Screen Shot 2022-12-02 at 02 07 36" src="https://user-images.githubusercontent.com/74501523/205180852-8ba3be70-4f07-4fa3-ac9d-7825de8353ed.png">

Origin design
<img width="461" alt="Screen Shot 2022-12-02 at 02 29 17" src="https://user-images.githubusercontent.com/74501523/205180885-3fe4ae6a-763f-42a8-a6e8-641d420f11c2.png">